docs/tutorials/basics/50_control_flow.md

```diff
@@ -240,3 +240,33 @@ end
 
 Both clauses have branches with the same conditions but in a different order and they behave differently.
 The first matching condition selects which branch executes.
+
+## Iteration
+
+Crystal offers several ways to do iteration. If iterating over an object, a natural way is to use `each`:
+
+```crystal-play
+vals = ["one", "two", "three", "four"]
+vals.each { |num| puts num }
+```
+
+The `loop` construct can also be used:
```

Comment:
I'd start with `while` first (which is indeed a construct), and then with `loop`, which is a [function](https://crystal-lang.org/api/1.3.0/toplevel.html#loop%28%26%29-class-method). In case it isn't clear, to me a construct is like a primitive, and `loop` isn't.

Reply:
@beta-ziliani Thanks for the review. Reordered and add a few further examples.  Further suggestions are welcome.